<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDossierParDef</class>
 <widget class="QDialog" name="DialogDossierParDef">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Actualisation MP3</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcourir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="actuCompil">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser Compil</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="actuAlbums">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser Albums</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="actuLives">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser Lives</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONFIGURATION DE L'ACTUALISER MP3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDossierParDef</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDossierParDef</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
